src/server/server.js

```diff
@@ -80,10 +80,12 @@ for (const middlewareUse of modules.middlewaresUse) {
   app.use(middlewareUse);
 }
 for (const middlewareGet of modules.middlewaresGet) {
-  if (middlewareGet.callback2) {
-    app.get(middlewareGet.path, middlewareGet.callback, middlewareGet.callback2);
-  } else {
-    app.get(middlewareGet.path, middlewareGet.callback);
+  if (!isEmpty(middlewareGet)) {
+    if (middlewareGet.callback2) {
+      app.get(middlewareGet.path, middlewareGet.callback, middlewareGet.callback2);
+    } else {
+      app.get(middlewareGet.path, middlewareGet.callback);
+    }
```

Comment:
This block is a bit hard to understand. Probably the naming convention is not very clear.

Reply:
Will refactor this part.